draft-ietf-mops-streaming-opcons.md

```diff
@@ -483,35 +485,39 @@ Here are a few common resolutions used for video content, with typical ranges of
 | 1080p (2K) | 1920 x 1080 | 6-8 Mbps | 4.5-7 Mbps
 | 2160p (4k) | 3840 x 2160 | N/A | 10-20 Mbps
 
+- Note that these codecs do not take the actual "available bandwidth" between streaming video servers and streaming video receivers into account when encoding, because the codec does not have any idea what network paths and network path conditions will carry the encoded video, at some point in the future.
+- Note that video receivers attempting to receive encoded video across a network path with insufficient available path bandwidth might request the video server to provide video encoded for lower bitrates, as described in {{adapt-deliver}}.
```

Comment:
This sentence seems to me it could use tightening up.  A suggestion:

Note also that media receivers often use the techniques from {{adapt-deliver}} to switch between these renditions according to what their networks can support.